src/bcs/serializable/move-structs.ts

```diff
@@ -439,7 +439,7 @@ export class MoveObject extends Serializable implements TransactionArgument {
     if (value instanceof AccountAddress) {
       this.value = value;
     } else {
-      this.value = AccountAddress.fromHexInput({ input: value });
+      this.value = AccountAddress.fromHexInputRelaxed({ input: value });
```

Comment:
Because I was getting objects from the API, it returned it with no leading 0, meaning I needed to sanitize my input. I then realized I don't see in any case why someone *wouldn't* want the "relaxed" version when using it as input.

You generally would only use a `MoveObject` to serialize an entry function argument, so the end serialization result is the same regardless of whether or not you input a `0x0...` or `0x...`, so `fromHexInputRelaxed` makes more sense to me than having people run into errors 1/16 times because they didn't expect the address they passed in to to fail the parsing test